wiki/Welcome/id.md

```diff
@@ -1,6 +1,6 @@
 # Selamat Datang
 
-Selamat datang di osu!, permainan ritme free-to-win terbaik yang dikembangkan oleh peppy dengan empat mode permainan yang dapat kamu nikmati di dalamnya: osu!standard, bagi para pemain yang ingin mengeklik lingkaran; osu!taiko, bagi para pemain yang ingin bermain drum; osu!catch, bagi para pemain yang ingin menangkap buah; dan osu!mania, bagi para pemain yang ingin menekan tuts-tuts keyboard mengikuti irama lagu.
+Selamat datang di osu!, game ritme free-to-win terbaik yang dikembangkan oleh peppy dengan empat mode permainan yang dapat kamu nikmati di dalamnya: osu!standard, bagi para pemain yang ingin bermain dengan lingkaran; osu!taiko, bagi para pemain yang ingin menabuh drum; osu!catch, bagi para pemain yang ingin menangkap buah; dan osu!mania, bagi para pemain yang ingin menekan tuts-tuts keyboard mengikuti irama lagu.
```

Comment:
bukannya gim?
```suggestion
Selamat datang di osu!, gim ritme free-to-win terbaik yang dikembangkan oleh peppy dengan empat mode permainan yang dapat kamu nikmati di dalamnya: osu!standard, bagi para pemain yang ingin bermain dengan lingkaran; osu!taiko, bagi para pemain yang ingin menabuh drum; osu!catch, bagi para pemain yang ingin menangkap buah; dan osu!mania, bagi para pemain yang ingin menekan tuts-tuts keyboard mengikuti irama lagu.
```

Reply:
ah, actually i changed `permainan` into `game` for this line in order to make it consistent with what has been written in the main page which is `game ritme free-to-win terbaik` o.o

![image](https://user-images.githubusercontent.com/36564236/94881344-69651a00-048f-11eb-9cc2-10fc2f52e99b.png)